src/actions/schemes.ts

```diff
@@ -50,24 +44,15 @@ export function createScheme(scheme: Scheme): ActionFunc {
 
 export function deleteScheme(schemeId: string): ActionFunc {
     return async (dispatch: DispatchFunc, getState: GetStateFunc) => {
-        dispatch({
-            type: SchemeTypes.DELETE_SCHEME_REQUEST,
-            data: null,
-        }, getState);
         let data = null;
         try {
             data = await Client4.deleteScheme(schemeId);
         } catch (error) {
             forceLogoutIfNecessary(error, dispatch, getState);
-            dispatch(batchActions([
-                {type: SchemeTypes.DELETE_SCHEME_FAILURE, error},
-                logError(error),
```

Comment:
@nadalfederer hey, Can you add the dispatch of logError back here? We still would like to keep those.